src/config.ts

```diff
@@ -33,6 +33,7 @@ export interface IProxyOption {
     enableOAS?: boolean;
     cors?: CorsOptions;
     // experimental options
+    enableAllEndpoint?: boolean;
```

Comment:
Is it an experimental option? If so, should it start with `exp`?

Reply:
I do not consider this an experimental option.

Reply:
Right! I see my comment might have been a bit unactionable. If it's not experimental, then it should probably be placed before the "experimental options" comment, so that it doesn't appear that way, I'd think. Doesn't impact the end user, but for making the code easier to understand it'd be a good idea, I think 😄